# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/notification/moz.build
# Commit: 93937caf6d13
# Full Hash: 93937caf6d13cd0d6ef427f3315e6a29c30a385c
# Author: Kagami Sascha Rosylight <saschanaz@outlook.com>
# Date: 2024-11-08 04:19:40
# Regressor Bug: 1928702
# File Overlap Count: 1
# Description:
#   Bug 1928702 - Part 1: Add skeleton PNotification r=asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D227686
# ==============================================================================

diff -r 785541f0311a -r 93937caf6d13 dom/notification/moz.build
--- a/dom/notification/moz.build	Thu Nov 07 17:07:15 2024 +0000
+++ b/dom/notification/moz.build	Thu Nov 07 17:32:03 2024 +0000
@@ -20,12 +20,22 @@
     "NotificationEvent.h",
 ]
 
+EXPORTS.mozilla.dom.notification += [
+    "NotificationChild.h",
+    "NotificationParent.h",
+]
+
 UNIFIED_SOURCES += [
     "Notification.cpp",
     "NotificationEvent.cpp",
+    "NotificationParent.cpp",
     "NotificationUtils.cpp",
 ]
 
+IPDL_SOURCES += [
+    "PNotification.ipdl",
+]
+
 include("/ipc/chromium/chromium-config.mozbuild")
 
 FINAL_LIBRARY = "xul"
